products/zoningtaxlots/models/intermediate/int__zoningdistricts.sql

```diff
@@ -147,17 +147,28 @@ new_order AS (
 lotzoneperorder AS (
```

Comment:
And sorry - I do think here it's also better not to pivot. These intermediate tables are useful for debugging